apps/dtlaunch/app-b2.js

```diff
@@ -1,28 +1,30 @@
+{ // must be inside our own scope here so that when we are unloaded everything disappears
+
 /* Desktop launcher
 *
 */
 
-var settings = Object.assign({
+let settings = Object.assign({
   showClocks: true,
   showLaunchers: true,
   direct: false,
   oneClickExit:false,
-  swipeExit: false
+  swipeExit: false,
 }, require('Storage').readJSON("dtlaunch.json", true) || {});
 
 if( settings.oneClickExit)
-  setWatch(_=> load(), BTN1);
+  setWatch(_=> returnToClock, BTN1);
```

Comment:
You need to store this in a variable and call `clearWatch` with it in `returnToClock` to remove it.

Reply:
Thanks! With that it seems there's no ram leak anymore :)